### ISSUE TITLE
Music skipping automatically

### ISSUE THREAD

**Issue by filipemanuelofs** [fast4x/RiMusic#2539]
### Steps to reproduce the bug

1. Playlist with at least 5 songs (one of which must be "[Impulse (feat. Devilish Trio) - Altare](https://www.youtube.com/watch?v=_Eq_qQUPvGQ)")
2. Leave it playing until you reach the song in question
_Do not interfere with the automatic process, fast forward or backward to that specific song, the bug doesn't happen. It only happens when music comes in automatically after a period of 2 to 3 songs playing without interference._

### Expected behavior

The music shouldn't skip.

### Actual behavior

When this song automatically comes on, 30 seconds long, it skips to another one, for no apparent reason.

### Screenshots/Screen recordings

I decided not to edit the video to show that the previous songs play normally, but when it comes to this specific one, it skips for no apparent reason. If you want, you can skip to 07:20 and see the behavior.

_Music is muffled because I had to reduce the video quality by **84%**._

https://github.com/fast4x/RiMusic/assets/1047292/2e75b2f8-5fe5-4717-8f92-ac022020723d

### RiMusic version

v0.6.41

### Android version

Android 14

### Additional information

tl;dr
I have a playlist and, when this song automatically comes on, 30 seconds long, it skips to another one, for no apparent reason. It took me a long time to catch this clip because I always thought I was bumping the headphone (bluethooth) or making some movement that was skipping the song, but I noticed the pattern of it always being this song and always lasting approximately 30s. Since then I've been trying to record and this time I succeeded.

**Comment by fast4x** [comment]
Switch to next occur if there is a playback error...
